src/listen.js

```diff
@@ -160,6 +160,11 @@ module.exports = function(defaultFuncs, api, ctx) {
 
               return globalCallback(null, fmtMsg);
               break;
+            case 'inbox':
+              // Could be message request
```

Comment:
It's kind of hard to reproduce the message request scenario.

v looks like this:
`
{ unread: 1,
  unseen: 1,
  seen_timestamp: 0,
  realtime_viewer_fbid: <fb_id>,
  type: 'inbox' }
`
I get these `v.type === 'inbox'` things whenever a message is received ( including message requests ).
I think there have to be some `v.type` that are for message requests only, but I haven't figure this thing out yet.